Step2

ArranHL opened the pull request:
### Step in progress

Step 2's objective is to have the VI's:
- Find the RPi with the SSH user
- Log in by Pi by the SSH key
- Add the Key to Github
- Add the Key to treehouses
- Add the key on RPI
- Log in by Root
- Rename Pi by Github Name
- Turn on Tor
- Turn on Tor Notice


I was going to have the VI's Enable SSH on your Raspberry Pi via a different way but thought that perhaps the simplest way may be the best.
There also needs to be a discussion about whether to make SSHing into the Pi more general for the sake of simplicity

dogi commented on _posts/2018-05-05-moodlemootph18.md:
```diff
@@ -1,7 +1,7 @@
 title: treehouses.io in MoodleMoot Philippines 2018
 layout: post
-author: mappuji
+author: empeje
```
@empeje ? please make an different issue/PR for both name changes they can be in one but not here ;)

empeje replied:
sure, done here https://github.com/treehouses/treehouses.github.io/pull/36

dogi commented on _posts/2019-01-27-try-treehouses-remote.md:
```diff
@@ -0,0 +1,100 @@
+Requirements:
+* Android Phone
+  - [ ] Install [treehouses/remote](https://github.com/treehouses/remote/releases) app
+* RaspberryPi 3B or higher (any version with bluetooth)
```
tested on
- 3B
- zero W (and WH)
- 3B+
- 3A+

empeje replied:
updated, please check @dogi

dogi commented on _posts/2019-01-27-try-treehouses-remote.md:
```diff
@@ -0,0 +1,100 @@
+* Android Phone
+  - [ ] Install [treehouses/remote](https://github.com/treehouses/remote/releases) app
+* RaspberryPi 3B or higher (any version with bluetooth)
+  - [ ] Install [treehouses/builder][GETTING_STARTED] image (in this example we use treehouses image version `75`)
```
- dev.ole.org or faster on cloudflare ... get our latest link here

empeje replied:
cloudfront you mean? what is the link @dogi?

dogi commented on _posts/2019-01-27-try-treehouses-remote.md:
```diff
@@ -0,0 +1,100 @@
+  - [ ] Install [treehouses/builder][GETTING_STARTED] image (in this example we use treehouses image version `75`)
+* A laptop
+  - [ ] Install [Etcher](https://www.balena.io/etcher/)
+  - [ ] Install Tor Browser (we have a fun demonstration)
```
:D

dogi replied:
we should add link to torbrowser download

empeje replied:
updated

ArranHL committed:
Step2

### Step in progress

Step 2's objective is to have the VI's:
- Find the RPi with the SSH user
- Log in by Pi by the SSH key
- Add the Key to Github
- Add the Key to treehouses
- Add the key on RPI
- Log in by Root
- Rename Pi by Github Name
- Turn on Tor
- Turn on Tor Notice


I was going to have the VI's Enable SSH on your Raspberry Pi via a different way but thought that perhaps the simplest way may be the best.
There also needs to be a discussion about whether to make SSHing into the Pi more general for the sake of simplicity by using

Liew211 committed:
Update firststeps.md

dogi committed:
Update firststeps.md

dogi commented:
@ArranHL ssh is already activated on treehouses image

Step 2's objective is to have the VI's:
- Log in by Pi user with password
- Rename Pi by Github Name
- create sshkeys
- Add the Key to Github
- Add the key on RPI by treehouses command
- Log in by Root
- Turn on Tor
- Turn on Tor Notice

Liew211 committed:
Update firststeps.md

dogi commented on pages/vi/firststeps.md:
```diff
@@ -145,26 +145,28 @@ In this step you will:
-To connect to Pi via SSH you will need to know your Raspberry Pi's local IP address. If you are running the Pi without a screen, you can find the IP address by opening your terminal and type `nmap -sP 192.168.1.*` to find your raspberri Pi's IP address 
+### Log in with the Pi user
 
+Once you start up your Raspberry Pi and find its local IP address, you can connect to it from your computer. Windows users can use an SSH client like PuTTY.  
```
git bash

dogi commented on pages/vi/firststeps.md:
```diff
@@ -145,26 +145,28 @@ In this step you will:
+To rename your Raspberry Pi, run `treehouses rename "username"` and replace `username` with your Github username.
+
 ## Tor
 Tor is a computer network run by volunteers worldwide. Each volunteer runs what is called a relay, which is just a computer that runs software allowing users to connect to the Internet via the Tor network.
```
we do not run a relay ... we are running a hidden service

Liew211 committed:
Update firststeps.md

Liew211 committed:
Create sshpi.md

Liew211 committed:
Update firststeps.md

Liew211 committed:
Update sshpi.md

Liew211 committed:
Update sshpi.md

Liew211 committed:
Update sshpi.md

dogi commented:
https://raw.githack.com/treehouses/treehouses.github.io/Step2/#!pages/vi/sshpi.md

Liew211 committed:
Update sshpi.md

empeje commented:
Hi @dogi I close this PR as I think things has changed in the Treehouses wiki.
